aiida_pseudo/groups/family/pseudo_dojo.py

```diff
@@ -107,6 +306,18 @@ def format_configuration_label(cls, configuration: PseudoDojoConfiguration) -> s
             pseudo_format=configuration.pseudo_format,
         )
 
+    @classmethod
+    def get_urls(cls, label):
+        """Get the pseudo archive and djrepo archive urls for a given configuration label.
+
+        :param label: PseudoDojoConfiguration label
+        :return: dictionary with keys 'pseudo' for pseudo archive and 'djrepo' for djrepo archive urls
+        """
+        urls = cls.urls.get(label)
```

Comment:
This will just except if `label` is not in the `urls` dictionary won't it? Should probably be `urls = cls.urls.get(label, None)`. But in that case it is better to change this to try/except